test/test-rotatesTetrominos.js

```diff
@@ -0,0 +1,32 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris');
+describe("rotate L-tetromino", function (){
+    class Tetramino = {
+        constructor(tetramino, phase, coordinates){
+            this.typeTetramoni = tetramino
+            this.initialRotationPhase = phase;
+            this.initialCoordinates = coordinates;
+        }
+        tetris.rotateTetromino(){
```

Comment:
This is class method. This is not components of tetris.
Also other methods. 
See how methods are defined in classes